demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -0,0 +1,232 @@
+        assert version_number == expected_version
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+
+    def test_bump_version_number_major(self):
+        """
+            Given:
+                - a pack name and version
+            When:
+                - bumping the version number in the metadata.json
+            Then:
+                - return the correct bumped version number
+        """
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'))
+        expected_version = '2.0.0'
+        from demisto_sdk.commands.update_release_notes.update_rn import UpdateRN
+        update_rn = UpdateRN(pack="HelloWorld", update_type='major', pack_files={'HelloWorld'})
+        update_rn.metadata_path = os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json')
+        version_number = update_rn.bump_version_number(pre_release=False)
+        assert version_number == expected_version
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+
+    def test_bump_version_number_revision(self):
+        """
+            Given:
+                - a pack name and version
+            When:
+                - bumping the version number in the metadata.json
+            Then:
+                - return the correct bumped version number
+        """
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'))
+        expected_version = '1.0.1'
+        from demisto_sdk.commands.update_release_notes.update_rn import UpdateRN
+        update_rn = UpdateRN(pack="HelloWorld", update_type='revision', pack_files={'HelloWorld'})
+        update_rn.metadata_path = os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json')
+        version_number = update_rn.bump_version_number(pre_release=False)
+        assert version_number == expected_version
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+
+    def test_bump_version_number_revision_overflow(self):
+        """
+            Given:
+                - a pack name and invalid version
```

Comment:
Is it in valid or is it valid and we just get overflow?
fix `Given` and `When` accordingly be specific.